### ISSUE TITLE
  "errorMessage": "Process exited before completing request"

### ISSUE THREAD

**Issue by lazukars** [ysugimoto/aws-lambda-image#35]
I've followed the instructions, have uploaded the .zip file to AWS lambda, and have tried to upload a sample image on S3 to see if it got resized. That did not work. No images were resized. So I then tried to test the lambda function via the "Lambda Management" console, which resulted in this error: 

`"errorMessage": "Process exited before completing request"`

When setting up the lambda function are there any specific settings that I have to choose or are required for your code to work? I've chosen the correct version of node.

Here is my config.json file: 

`{
  "bucket": "example.guide",
  "reduce": {
      "directory": "images"
  },
  "resizes": [
    {
      "size": 300,
      "directory": "med"
    }
  ]
}`

So basically, if I upload an image to example.guide/images/ I'd expect your script to resize the image and place it in example.guide/images/med or example.guide/med. It does neither.

**Comment by odedniv** [comment]
This error means the code crashed at some point and didn't reach the end of it.

You should examine the log, either in the Lambda screen (when you run the code using the "Test" button), or from CloudWatch. Post the stacktrace/error or try to narrow down what's wrong with your configuration/bucket.

**Comment by timdinh-grex** [comment]
I got the same error. After 2 days of troubleshooting i found out that the process crashed if you include the nuget package Microsoft.IdentityModel.Tokens or any packages that reference that. 
The version  was using was 5.1.3.
I reckon it may have been a binary compatibility issue to have the whole process crash like that.
